panel/src/components/Forms/FormControls.vue

```diff
@@ -1,25 +1,29 @@
 <template>
-	<k-button-group
-		v-if="buttons.length"
-		layout="collapsed"
-		class="k-form-controls"
-	>
-		<k-button
-			v-for="button in buttons"
-			:key="button.text"
-			v-bind="button"
-			size="sm"
-			variant="filled"
-		/>
+	<div v-if="buttons.length" class="k-form-controls">
```

Comment:
`Question:` Why isn't the `k-button-group` an option as wrapper element anymore? Would save a div level.

Reply:
When the dropdown is the last element, the collapsed button group creates a weird display CSS glitch. The rounded corners on the last element vanish when you open the dropdown.